[ARCHIVED] TeradataScripts/Teradata/bin/create_ddls.sh

```diff
@@ -0,0 +1,120 @@
+#
+#Version 20201105: Script created
+#Version 20211210: Updated to fix error messages 
+
+
+##### Modify the connection information
+connection_string="127.0.0.1/dbc,dbc"
```

Comment:
In the other Create_DDL file the IP was removed from the connection string. Should we do the same here?

Reply:
As you can see this is another folder where the old scripts are archived
This folder is not accesible to the customer, but it was a Najeeb's recommendation to keep the old files files archived